src/internal/deploy/cmds/cmds.go

```diff
@@ -64,14 +65,8 @@ access, you can check progress with:
 
   kubectl get pod -l release=pachyderm-ide
 
-Once all of the pods are in the 'Ready' status, you can access the IDE in the
-following manners:
-
-* If you're on docker for mac, it should be accessible on 'localhost'.
-* If you're on minikube, run 'minikube service proxy-public --url' -- one or
-  both of the URLs printed should reach the IDE.
-* If you're on a cloud deployment, use the external IP of
-  'kubectl get service proxy-public'.
+Once all of the pods are in the 'Ready' status, you can access the IDE
+by running 'pachctl port-forward' and visiting 'localhost:30659'
```

Comment:
Looking at the docs for the helm chart, they do have you deploy a load-balanced service with no TLS and no authenticator to the cloud by default 🤕 

My thinking about this is that we probably shouldn't make that foot-gun case easy? It would be hard anyways because you'd need to know the redirect URI, which will be the random load-balancer IP. I think for a cloud deployment the minimum standard is going to be:

- be able to assign a hostname or fixed IP for the service
- some kind of TLS

It sucks that deploying a web-accessible service in k8s is not straightforward, but the alternative is that we're telling customers to put their data on the internet with no security, which doesn't seem great.